src/pages/documentation/getting-started/get-the-sdk.md

```diff
@@ -28,18 +38,6 @@ iOS
 
 <Tabs query="platform=flutter&task=get"/> -->
 
-<!-- Cordova
-
-<Tabs query="platform=cordova&task=get"/>
-
-Unity
-
-<Tabs query="platform=unity&task=get"/>
-
-Xamarin
-
-<Tabs query="platform=xamarin&task=get"/>  -->
-
```

Comment:
Are we planning to have Cordova/Unity/Xamarin support again in the future? If not, we can just delete this.

Reply:
Yup, let us just delete it.